app/src/main/res/values/speakers_summary_layout_attrs.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="SpeakersSummaryLayout">
+        <attr name="max_icon_count" format="integer" />
+        <attr name="image_size" format="dimension" />
+        <attr name="textColor" format="color" />
+    </declare-styleable>
+    <integer name="default_speakers_summary_layout_max_icon_count">5</integer>
+    <dimen name="default_speakers_summary_layout_image_size">28dp</dimen>
+    <string name="speakers_summary_layout_image_content_description">Speaker image</string>
```

Comment:
I think it's better to divide them to strings.xml and dimens.xml and integers.xml.
Especially strings.xml, it's easier to find the resources in the project.
What do you think?

Reply:
@konifar I agree to your opinion. 
I believe some people like the mixed resource definition, but it is not general at all. It would be better to use general way in the conference-app development.
I will change my code. Thank you!

Reply:
Yes, that's what I wanted to do! Thanks!